data/configurables/province_terrains.txt

```diff
@@ -8874,64 +8874,80 @@ xDC9333="plains"
 x080E16="plains"
 x0D7B7C="plains"
 xB7B43B="plains"
-x939B64="plains"
-x6A0BEC="plains"
-x06C646="plains"
 x28E8B3="plains"
 x81E2C0="plains"
 x8C14B7="plains"
 xF77148="plains"
 x0E5076="plains"
 x454DCA="hills"
-x04A22A="plains"
-x049ECA="plains"
 xF51380="plains"
-x1ABE9A="plains"
-xD569C6="plains"
 x41FBE5="plains"
 xC2624F="plains"
-x65E38A="plains"
-x0BEBF2="plains"
 x1D529E="plains"
 xE1756B="hills"
-x498F40="plains"
 x90E2A0="plains"
 x765D85="plains"
-x01950F="plains"
 xF03A34="plains"
-x2B0F26="plains"
 xC5E68C="plains"
-x8FD594="plains"
-x92711A="plains"
 xB0A559="plains"
 xF10966="plains"
+xD69D1F="plains"
+x571DFC="plains"
+x020E07="plains"
+x30DDF1="plains"
+x4DB84A="plains"
+x6892D3="plains"
+xBC9607="plains"
+xE13F68="plains"
+x7D6986="hills"
+x557928="plains"
+xF4C230="plains"
+xE8B11C="plains"
+x1842A7="plains"
+x2F3C33="plains"
+x1347CA="plains"
+x60D100="plains"
+xFCDC34="plains"
+xA48A17="plains"
+x573287="plains"
+x913950="plains"
+x8FBFEA="plains"
+x50B7B1="plains"
+xAFA8BA="plains"
+x90C920="plains"
+xA05180="plains"
+x939B64="plains"
```

Comment:
This and following are off by about 1000 (i.e. x939B64 has game id of 9290); xA05180 and earlier are correct